base/uk.ac.stfc.isis.ibex.configserver/src/uk/ac/stfc/isis/ibex/configserver/editing/EditableConfiguration.java

```diff
@@ -298,7 +329,7 @@ public Group apply(EditableGroup group) {
      * @return The IOCs associated with the configuration
      */
     private Collection<Ioc> getIocs() {
```

Comment:
This method is used when transforming an EditableConfiguration to a Configuration (the format in which it is then sent to the blockserver), to populate that new Configuration object with the right types of elements - same as getBlocks, getGroups getComponents in this class. I can rename those methods to transformIocs etc. to more accurately reflect the purpose of the methods. Does that sound sensible?